_posts/0000-01-02-anthonykiefer.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
https://i.chzbgr.com/original/8944133/h126EBC4E/17-coding-memes-for-the-frustrated-software-engineer